### DISCUSSION TITLE
get sample rate  , channels , sample  bits as variable from decoder .

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#921] [Q&A] [answered]
decoder successfully change i2s sample rate  ,bits and channel   --- decoder.setNotifyAudioChange(i2s); -- 

but i want to display these attribute on screen so i need it as variable   . 
 
 I read relevant documents but cant find the solution nor in any example . so need your guidance  .

**Comment by pschatzmann** [comment, marked as answer]
If you check the class documentation you will find the audioInfo() method!
